index.md

```diff
@@ -1,30 +1,39 @@
-Anbox Cloud offers a software stack that runs Android applications in any cloud enabling high-performance streaming of graphics to desktop and mobile client devices.
+Anbox Cloud provides a rich software stack that enables you to run Android applications in the cloud for all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices.
 
-At its heart, it uses lightweight container technology instead of full virtual machines to achieve higher density and better performance per host while ensuring security and isolation of each container. Depending on the target platform, payload, and desired application performance (e.g. frame rate), more than 100 containers can be run on a single machine.
+Using container technology, Anbox Cloud is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. It is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).
 
-For containerization of Android, Anbox Cloud uses the well established and secure container hypervisor [LXD](https://linuxcontainers.org/). LXD is secure by design, scales to a large number of containers and provides advanced resource management for hosted containers.
+## Core features
 
-Also have a look at the [official Anbox Cloud website](https://anbox-cloud.io/) for more information.
+* Simple and straightforward deployment
+* Management of the container and application life cycle while optimising for high density, performance and fast container boot times
+* Platform integration tools to allow, for example, integration of existing streaming solutions
+* Support for both x86 and Arm64 hardware
+* Integration with 3rd party solutions for binary translation solutions on Arm64-only hardware
```

Comment:
I don't understand what we're saying here ... any way to make it clearer?

Reply:
Lets drop this item. It's not true anymore.